CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Multithreading support (SMP)
 - Added support for code:load_abs/1, code:load_binary/3
 - Added support for loading / closing AVMPacks at runtime
+- Added support for building with ESP-IDF v5
```

Comment:
We are already building on esp-idf v5.x so I would say something different, since this change makes the v5 build in feature parity with the v4.
I suggest: `Support for ESP-IDF v5.x`